tests/test_grpc.py

```diff
@@ -93,6 +93,32 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
+    
+    ip='127.0.0.2'
+    mapdl = launch_mapdl(ip=ip)
+
+    # Check the ip
+    # print(mapdl._ip)
+
+    # Check it also in the output file.
+    # print(mapdl.directory)
+
+    output_file = mapdl._download_as_raw('.__tmp__.out')
+    output = output_file.decode().splitlines()
+
+    # Catching the line where the IP of the gRPC server is specified.
+    for each_line in output:
+        if 'Server listening on' in each_line: 
+            line = each_line
+            print(each_line)
+
+    mapdl.exit()
+
+    assert ip in line
```

Comment:
This is a good place to use regular expressions:

```py
import re

output = """
 ##############################
 ### START GRPC SERVER      ###
 ##############################

 Server Executable   : MapdlGrpc.Server
 Server listening on : 0.0.0.0:50052
"""

ip = '0.0.0.0'
finds = re.findall('listening on : (.*):', output)
assert finds[0] == ip
```

Regular expressions are hard learn, but quite powerful.  I generally play around with an online tool like https://regex101.com/ and then copy/pasta into python when finally using it.  Way better (and faster) than searching line by line.

Reply:
I totally agree that regex is powerful. I have used in the past for quite a few things. Lastly I tried to avoid it because, either the people didn't understand it or (even worse) I tend to forget how it worked xD. 

But if you are happy with using regex, I will use it. No problem.

Reply:
Replaced in the next commit.